decidim-core/app/packs/src/decidim/sw/loader.js

```diff
@@ -0,0 +1,16 @@
+// check if the browser supports serviceWorker at all
+window.addEventListener("load", async () => {
+  if ("serviceWorker" in navigator) {
+    // eslint-disable-next-line no-unused-vars
+    const registration = await navigator.serviceWorker.register("/sw.js", { scope: "/" });
+    const permission = await window.Notification.requestPermission();
```

Comment:
Is this necessary for offline mode?

Reply:
Besides, tests fail because of this 
<img width="1438" alt="Screenshot 2021-12-20 at 05 22 46" src="https://user-images.githubusercontent.com/17616/146711451-1d123785-3348-43cb-9cca-b29c10a55a4a.png">

Reply:
Snippet commented out